### ISSUE TITLE
MA0151 FP incorrectly parsing format string

### ISSUE THREAD

**Issue by Bouke** [meziantou/Meziantou.Analyzer#767]
### Version of the Meziantou.Analyzer NuGet package

2.0.176

### Rule Identifier

MA0151

### Target Framework

net8.0

### C# Language version

C# 12

### Description

The format string can contain escaped brackets `\{`, but that's incorrectly parsed as being a placeholder.

### Reproduction Steps

Minimal code:

```c#
[DebuggerDisplay(@"Person \{ Name = {Name} \}")]
public class Person(string Name);
```

> Member ' Name = {Name' does not exist

**Comment by meziantou** [comment]
Fixed by 32732ae51c09918a147bb378c9249e306e6c0953. The new NuGet package will be available in a few minutes.

**Comment by Bouke** [comment]
Thank uou!
